### DISCUSSION TITLE
Knack Add Filter Sorting

### DISCUSSION THREAD

**Discussion by jbk68** [cortexrd/Knack-Toolkit-Library#196] [General]
I know that using the keyword _nf will hide fields from being shown when a user uses Knack's "add filter".  But I was wondering if there is currently a way to sort the way the fields are shown when "add filter" is used.  Currently the fields appear to be shown as they are organized on the table from left to right.  I'd like to have the ability to sort based on names A>Z or Z<A are even better,  be able to custom sort them in any way I want a user to see them.  Maybe the ones I feel are the most popular on top.  ???

**Comment by LeighBerrell** [comment]
Maybe just rearrange your table so the fields are in the sort order you need?

**Comment by CSWinnall** [comment]
That is what we did.  The order of fields in the table will determine the order of fields in the filter.  The only thing we couldn't rearrange is the groupings.  The grouped field will always be first.

**Reply by jbk68** [reply]
Hi Craig and thank you for your reply.  I'm aware that how your table is arranged will determine the order of the fields in the filter, but I would like to be able to sort them differently on the filtered list in some cases then the order they appear on the table.  For example more  popular fields listed first or even A>Z or Z>A.  If it's possible that would be great, if not, no big deal.
